modules/node/src/channel/channel.entity.ts

```diff
@@ -88,3 +89,58 @@ export class Channel {
   @UpdateDateColumn()
   updatedAt: Date;
 }
+
+export interface ChannelJSON {
+  multisigAddress: string
+  schemaVersion: number
+  addresses: CriticalStateChannelAddresses
+  userIdentifier: string
+  nodeIdentifier: string
+  available: boolean
+  activeCollateralizations: Collateralizations
+  appInstances: AppInstanceJSON[]
+  monotonicNumProposedApps: number
+  // withdrawalCommitments: WithdrawCommitmentJSON[]
```

Comment:
These fields are not queried via `cfCore`, but should be included for the sake of completeness. I'll add them before this gets merged.

Reply:
See earlier comment about `StateChannelJSON` interface in the `types` module. We should try to extend that interface and add the node-specific params.

Reply:
I'll see if I can plug this directly into the protocol.